mkosi/distributions/fedora.py

```diff
@@ -159,9 +159,10 @@ def invoke_dnf(state: MkosiState, command: str, packages: Iterable[str], env: Ma
     state.workspace.joinpath("vars").mkdir(exist_ok=True)
 
     cmdline = [
-        'dnf' if shutil.which('dnf') else 'yum',
+        'dnf5' if shutil.which('dnf5') else ('dnf' if shutil.which('dnf') else 'yum'),
```

Comment:
If we call `shutil.which` we can directly use the result, since it's `None` if what we're looking for doesn't exist, which makes this a bit shorter.
```suggestion
        shutil.which("dnf5") or shutil.which("dnf"') or "yum"
```

Reply:
Updated like that.